src/SUnit-Core/BadEqualer.class.st

```diff
@@ -8,7 +8,9 @@ Class {
 }
 
 { #category : #comparing }
-BadEqualer >> = other [ 
+BadEqualer >> = other [
```

Comment:
Hmm, this class seems unused. I wonder if it's useful.

Next to it, `BadHasher` seems to be used in `HashTesterTest` and `HashTester` to do some kind of fuzzing of hashes.

In that tag (`SUnit-Core-Utilities`) there seem to be many classes that are not utilities.

Maybe to attack in another issue.

Reply:
Yess BadEqualer is not used yet. I plan to write some tests in SUnit package.
For the seond point (utilities), i will take a deep look on.